### ISSUE TITLE
Last update of the cheat 10 days ago

### ISSUE THREAD

**Issue by ddjem87** [Teklad/csgo-horker#51]
Hey i saw the cheat does not have any updates is there something wrong with it?
is it still save to play with it?

**Comment by kazukeno** [comment]
Yes its safe but i don't know why teklad isn't updating it. It's maybe that the overlay update is hard to make. He will probably publish the update as soon as possible.

**Comment by ddjem87** [comment]
alright i was kinda scared cause before 1/2 weeks there were updates every 2/3 days thank you anyways.

**Comment by Teklad** [comment]
I was a little busy with some other projects.  I'm also rewriting a lot of the base components to be more programmer-friendly.

TL;DR  I got tired of seeing if(Read) everywhere in my code.

**Comment by ddjem87** [comment]
i am just asking is this your overlay project?

**Comment by ddjem87** [comment]
i am just asking is this your overlay project cause there is github link for this cheat

https://www.youtube.com/watch?v=rgHxi0GqpP4

**Comment by Teklad** [comment]
No, the overlay project is a separate entity all together.  I'm working on a simplistic overlay window system that's going to allow me to put settings and stuff on top of your CSGO window.  It **should** be very similar to using an internal in terms of looks.

It won't be too advanced at first, just your basic sliders and checkboxes for interacting with settings.  In the long run I'm hoping to have workable scrolling, drop-downs, and others on top of drawing primitives for ESP, hit marker, etc.

The overlay is actually more keen to a library that programmers can use to integrate UI into their own externals.

**Comment by juhssi3** [comment]
Any estimate when its ready? :)

**Comment by Teklad** [comment]
Unfortunately, no.  I'd rather get it right the first time than have to rewrite it later on so I'm taking my time planning out the design so you guys get the best performance possible right out of the gate. The wait will be worth it, trust me.

**Comment by juhssi3** [comment]
ok :+1:
